src/screens/ImportSeedPhraseSheet.js

```diff
@@ -233,9 +232,15 @@ export default function ImportSeedPhraseSheet() {
       try {
         setBusy(true);
         setTimeout(async () => {
-          const { hdnode, isHDWallet, type, wallet } = getWallet(input);
-          setCheckedWallet({ hdnode, isHDWallet, type, wallet });
-          const ens = await web3Provider.lookupAddress(wallet?.address);
+          const type = identifyWalletType(input);
+          let walletResult;
+          if (type === WalletTypes.privateKey) {
```

Comment:
Can we just use the ethereumUtils.js `deriveAccountFromMnemonicOrPrivateKey` function here in this case?